src/pollution_model/run_medusa_2.py

```diff
@@ -28,33 +30,161 @@
 log = logging.getLogger(__name__)
 
 
+def read_roof_surface_polygons(engine: Engine) -> gpd.GeoDataFrame:
```

Comment:
```suggestion
def read_roof_surface_polygons(engine: Engine) -> None:
```
Return-type was incorrect